packages/near-membrane-base/src/blue.ts

```diff
@@ -57,10 +57,12 @@ function createBlueShadowTarget(target: BlueProxyTarget): BlueShadowTarget {
     if (typeof target === 'function') {
         // this new shadow target function is never invoked just needed to anchor the realm
         try {
+            // istanbul ignore next
             // eslint-disable-next-line func-names
             shadowTarget = 'prototype' in target ? function () {} : () => {};
         } catch {
             // target is a revoked proxy
+            // istanbul ignore next
             // eslint-disable-next-line func-names
             shadowTarget = function () {};
```

Comment:
☝️ And this one.

Reply:
Same line 58 comment, but I think I want to add the explicit comments now.